### PR TITLE
Modals composition api part2

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

**Comment by bonustrack** [comment]
@ChaituVR Can you check if everything working as usual with those changes?

File: src/components/Modal/Terms.vue [hunk 1]
```diff
@@ -34,18 +34,15 @@
...
-  computed: {
-    getIpfsUrl() {
-      return getUrl(this.space.terms);
+  setup(props, { emit }) {
+    function accept() {
+      emit('accept');
+      emit('close');
     }
+
+    return { accept, getIpfsUrl: getUrl(props.space.terms) };
```
Comment (ChaituVR) [inline src/components/Modal/Terms.vue:45]: In a proposal when space doesn't have a terms field, getting this error (ex: yam space / gitcoingdao space)
image

**Review by ChaituVR — COMMENTED** [review]
Looks like everything is good

File: src/components/Modal/Terms.vue (hunk 1 above)
Comment (samuveth) [reply src/components/Modal/Terms.vue:45]: Ah nice spot

**Comment by samuveth** [comment]
@ChaituVR please approve when done reviewing

**Review by ChaituVR — APPROVED** [review]
Everything looks to be working well

**Pull request merged**